plugins/plugin-kiali/src/components/panel/Graph.tsx

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React, { memo, useCallback, useEffect, useRef, useState } from 'react';
+import CytoscapeComponent from 'react-cytoscapejs';
+import cytoscape from 'cytoscape';
```

Comment:
Can you add the following import please:
```suggestion
import cytoscape from 'cytoscape';
import nodeHtmlLabel from 'cytoscape-node-html-label';
```